ui/values/strings.go

```diff
@@ -1032,6 +1033,13 @@ const (
 	StrWalletsNeedToSyncMsg            = "walletsNeedToSyncMsg"
 	StrIWillSyncLater                  = "iWillSyncLater"
 	StrOkaySync                        = "okaySync"
+	StrBackupDEXSeed                   = "backupDEXSeed"
+	StrDEXSeed                         = "dexSeed"
+	StrNewDEX                          = "newDex"
+	StrRestoreDEX                      = "restoreDEX"
+	StrRestoreDEXMsg                   = "restoreDEXMsg"
+	StrRestorationSeed                 = "restorationSeed"
+	StrOptionalRestorationSeed         = "optionalRestorationSeed"
```

Comment:
I only see `backupDEXSeed`, `dexSeed` and `optionalRestorationSeed` in the en.go file.

Reply:
updated